js/binance.js

```diff
@@ -6053,6 +6054,57 @@ module.exports = class binance extends Exchange {
         };
     }
 
+    async createMarginLoan (code, symbol, amount, params = {}) {
```

Comment:
In this case the `symbol` argument is required by the method signature, but is allowed to be undefined in the body of the method implementation, so we need to sync that, probably, by doing this everywhere:

```suggestion
    async createMarginLoan (code, amount, symbol = undefined, params = {}) {
```

Let me know your thoughts.

Reply:
Yeah this makes sense to me, let me know if there is any other changes to these new methods that you think might need to be added or changed

Reply:
Amount is almost always defined and uses `this.amountToPrecision(symbol, amount)` 
Should I only defined amountToPrecision when symbol is defined or is there a different approach?

Reply:
We should use currencyToPrecision instead of amountToPrecision in such cases.